configure.ac

```diff
@@ -389,6 +389,10 @@ if test x$GCC = xyes -a x$intel_compiler != xyes -a x$qcc_compiler != xyes -a x$
   LIBCRTS="-lgcc_s"
 fi
 
+AC_MSG_CHECKING([if compiler supports -march=native])
+AM_CONDITIONAL([COMPILER_SUPPORTS_MARCH_NATIVE], ["$CC" -march=native -S -o /dev/null -xc /dev/null > /dev/null 2>&1])
```

Comment:
This should use AC_COMPILE_IFELSE because there is often more to invoking the compiler than just $CC.